README.md

```diff
@@ -2,23 +2,27 @@
 > expect(element).to.have.xpath('//blink')
 
 ## Installation
+```bash
+> npm install --save-dev chai-have-xpath
 ```
-npm install --save-dev chai-have-xpath
+
+## Development
+```bash
+> npm install
+> npm run watch
 ```
 
 ## Usage
 
 ```javascript
-var chai = require('chai');
```

Comment:
Please do not use es6 in this readme as it does not have any information about es6